web/client/utils/MapInfoUtils.js

```diff
@@ -90,20 +77,86 @@ const MapInfoUtils = {
+     * @param {number} minX Minimum X.
+     * @param {number} minY Minimum Y.
+     * @param {number} maxX Maximum X.
+     * @param {number} maxY Maximum Y.
+     * @param {Object} optExtent Destination extent.
+     * @return {Object Extent.
+     */
+    createOrUpdate(minX, minY, maxX, maxY, optExtent) {
+        if (optExtent) {
+            optExtent.maxx = maxX;
+            optExtent.maxy = maxY;
+            optExtent.minx = minX;
+            optExtent.miny = minY;
+            return optExtent;
+        }
+        return { maxx: maxX, maxy: maxY, minx: minX, miny: minY };
+    },
+    /**
+     * Creates a bbox of size dimensions areund the center point given to it given the
+     * resolution and the rotation
+     * @param center {object} the x,y coordinate of the point
+     * @param resolution {number} the resolution of the map
+     * @param rotation {number} the optional rotation of the new bbox
+     * @param size {object} width,height of the desired bbox
+     * @param opt_extent {object}  optional bbox if passed it will be updated
+     * @return {object} the desired bbox {minx, miny, maxx, maxy}
+     */
+     getForViewAndSize(center, resolution, rotation = 0, size, optExtent) {
+        let dx = resolution * size[0] / 2;
+        let dy = resolution * size[1] / 2;
+        let cosRotation = Math.cos(rotation);
+        let sinRotation = Math.sin(rotation);
+        let xCos = dx * cosRotation;
+        let xSin = dx * sinRotation;
+        let yCos = dy * cosRotation;
+        let ySin = dy * sinRotation;
+        let x = center.x;
+        let y = center.y;
+        let x0 = x - xCos + ySin;
+        let x1 = x - xCos - ySin;
+        let x2 = x + xCos - ySin;
+        let x3 = x + xCos + ySin;
+        let y0 = y - xSin - yCos;
+        let y1 = y - xSin + yCos;
+        let y2 = y + xSin + yCos;
+        let y3 = y + xSin - yCos;
+        let bbox = MapInfoUtils.createOrUpdate(
+            Math.min(x0, x1, x2, x3), Math.min(y0, y1, y2, y3),
+            Math.max(x0, x1, x2, x3), Math.max(y0, y1, y2, y3),
+            optExtent);
+        return bbox;
+    },
     buildIdentifyRequest(layer, props) {
-        const {bounds, crs} = reprojectBbox(props.map.bbox, props.map.projection);
+        /* In order to create a valid feature info request
+         * we create a bbox of 101x101 pixel that wrap the point.
+         * center point is repojected then is built a box of 101x101pixel around it
+         */
+        let wrongLng = props.point.latlng.lng;
+        // longitude restricted to the -180,180 range
+        let lngCorrected = wrongLng - (360) * Math.floor(wrongLng / (360) + 0.5);
+        const center = {x: lngCorrected, y: props.point.latlng.lat};
+        let centerProjected = CoordinatesUtils.reproject(center, "EPSG:4326", "EPSG:900913");
+        const resolution = MapUtils.getCurrentResolution(props.map.zoom, 0, 21, 96);
+        const rotation = 0;
+        const size = [101, 101];
```

Comment:
what are you referring? if u mean reprojectBbox, is no longer used so it has been removed.